CGPs/0015.md

```diff
@@ -0,0 +1,39 @@
+# CGP [0015]: Extend Attestation Expiration Duration
+
+- Date: 2020-11-13
+- Author(s): @aslawson
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+This proposal will extend the expiration of a requested attestation from 1 HOUR -> 1 DAY.
+
+The attestation flow includes decentralized attestation services run by Validators. An on-chain request for an attestation randomly selects an issuer (instance of the service). The caller can then make an https requests to that service to trigger an SMS confirmation code. This code is used to verify that the caller has access to the phone number provided.
+
+The initial expiration time was selected at 1 hour with the assumption that it would be sufficiently long in duration for users to request and receive SMS deliveries of their verification codes and complete the attestation. After the expiration duration, it was assumed that an issue occurred requesting with the current issuer, thus it would make more sense to request a new issuer. However in practice, SMS may be delivered much later than anticipated. Even if delivered within the hour, a user may have already given up on the process with intent to returning to it later in the day. Furthermore, it is difficult for Valora wallet support to help users with reported user verification failures if the attestation has already expired. This change is expected to reduce the number of incomplete attestations, thus improving validator attestation success rates.
+
+## Proposed Changes
+
+Fill out the following template for each transaction in the proposal
+
+1. Set Attestation attestationExpiryBlocks to 60 * 60 * 24 / 5 (24 hours with 5 seconds per block)
+  - Destination: Attestations [setAttestationExpiryBlocks](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/identity/Attestations.sol#L518)
+  - Data: 17280
+  - Value: 0
```

Comment:
What is Value here? Is it related to "attestationExpiryBlocks has to be greater than 0"?

Reply:
`value` is CELO that would be sent with the transaction.  In this case, none would be needed.